packages/swingset-liveslots/src/collectionManager.js

```diff
@@ -366,6 +366,8 @@ export function makeCollectionManager(
 
     function deleteInternal(key) {
       mustMatch(key, keyShape, invalidKeyTypeMsg);
+      if (passStyleOf(key) === 'remotable' && getOrdinal(key) === undefined)
+        throw Fail`key ${key} not found in collection ${q(label)}`;
```

Comment:
```suggestion
      if (passStyleOf(key) === 'remotable' && getOrdinal(key) === undefined) {
        throw Fail`key ${key} not found in collection ${q(label)}`;
      }
```

Don't our lint rules already prevent this? If not, can we get them to?